RELEASE_NOTES.md

```diff
@@ -9,6 +9,8 @@ The `chef generate` command has been updated to produce cookbooks and repositori
   - `chef generate cookbook` now generates a cookbook with a Policyfile and no Berksfile by default. You can revert to the previous behavior with the `--berks` flag.
   - `chef generate cookbook` now includes ChefSpecs that utilize the ChefSpec 7.3+ format. This is a much simpler syntax that requires less updating as older platforms are deprecated.
   - `chef generate cookbook` no longer creates cookbbook files with the unecessary `frozen_string_literal: true` comments.
+  - `chef generate cookbook` no longer generates a full Workflow (Delivery) build cookbook by default. A new `--workflow` flag has been added to allow generating the build cookbook.
+  - `chef generate cookbook` no longer includes the unused `--delivery` flag. If you would like to generator a cookbook with a build cookbook use `--workflow` instead.
```

Comment:
If you would like to generate a cookbook with a build cookbook use `--workflow` instead.

Reply:
I tweaked this all a bit